editor.planx.uk/src/pages/FlowEditor/lib/store/team.ts

```diff
@@ -157,4 +158,12 @@ export const teamStore: StateCreator<
   setTeamMembers: async (teamMembers: TeamMember[]) => {
     set(() => ({ teamMembers }));
   },
+  deleteUser: async (userId: number) => {
+    const { $client } = get();
+    const response = await $client.user.delete(userId);
+    if (!response) {
+      throw new Error("Unable to remove user");
+    }
```

Comment:
It would be more valuable to catch the actual error here - in isolation if Airbrake had "Unable to remove user" it wouldn't be that easy to understand the root cause.

Reply:
Will follow up with TryCatch